templates/UserManagement/Profile/profile.html.twig

```diff
@@ -175,6 +184,17 @@
        data-number-of-following="{{ following_list|length }}"
   ></div>
 
+  <div class="js-user-share"
+       data-theme-display-name="{{ getThemeDisplayName() }}"
+       data-trans-check-out-project="{{ "checkOutProject"|trans({}, "catroweb") }}"
```

Comment:
"Checkout project" seems not to fit the current context. Please add a new translation.